README.rst

```diff
@@ -2290,8 +2290,8 @@ Configure timeout for the TCP connect when establishing a SSH connection.
 |              |                                                                                           |
 |              |   (                                                                                       |
 |              |     ('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),                         |
-|              |     ('openwisp_controller.connection.connectors.snmp.Snmp', 'OpenWRT SNMP'),              |
-|              |     ('openwisp_controller.connection.connectors.airos.snmp.Snmp', 'Ubiquiti AirOS SNMP'), |
+|              |     ('openwisp_controller.connection.connectors.openwrt.snmp.OpenWRTSnmp', 'OpenWRT SNMP'),              |
+|              |     ('openwisp_controller.connection.connectors.airos.snmp.AirOsSnmp', 'Ubiquiti AirOS SNMP'), |
```

Comment:
please readapt the restructured syntax of the other rows

Reply:
I'm not good with `.rst`. Let me try again :)

Reply:
the other lines have to be made of the same length so the table looks consistent

Reply:
> the other lines have to be made of the same length so the table looks consistent

Yes, just noticed in my editor it is looks consistent, but here it is scattered. See below: 
![Screenshot from 2022-05-18 00-21-51](https://user-images.githubusercontent.com/56113566/168888909-edfe16e1-97cd-4bc0-8f8c-62b8b911a344.png)
![Screenshot from 2022-05-18 00-22-12](https://user-images.githubusercontent.com/56113566/168888919-e2a102a5-f17d-4449-947e-4fe4acfd1f4b.png)